capstone/src/main/webapp/explore-loader.js

```diff
@@ -43,6 +43,11 @@ async function loadListings (clubs, studentClubs, interestedClubs) {
     template.content.querySelector('#club-name').innerHTML = club.name;
     template.content.querySelector('#club-name').href = 'about-us.html?name=' + club.name;
     template.content.querySelector('#description').innerHTML = club.description;
+    if (club['exclusive']) {
+      template.content.querySelector('#exclusive').innerHTML = 'Exclusive';
+    } else {
+      template.content.querySelector('#exclusive').innerHTML = '';
+    }
```

Comment:
nit: I think this could be cleaner using the ternary operator so something like:
template.content.querySelector('#exclusive').innerHTML = club['exclusive'] ? 'Exclusive' : '';
This should work but if not, it's fine :)

Reply:
Thanks! Not to familiar with ternary lol but this is a lot cleaner you're right